### ISSUE TITLE
expectNot() should fail when condition is true (#v2)

### ISSUE THREAD

**Issue by psaxton** [vlucas/frisby#351]
As a tester I use the expectNot function to test that an expectation should be false.
Currently expectNot succeeds whether the expectation is true or false.
It should fail when the expectation is true.

The following test demonstrates the issue:
```
'use strict';

const frisby = require('../src/frisby');
const mocks = require('./fixtures/http_mocks');

const testHost = 'http://api.example.com';

describe('Frisby expectNot()', function() {

  it('should fail when condition is true', function (doneFn) {
    mocks.use(['getUser1']);
    let failedExpectation;

    frisby.get(testHost + '/users/1')
      .catch(err => failedExpectation = err)
      .expectNot('status', 999)
      .then(() => expect(failedExpectation).toBeDefined())
      .done(doneFn);
  });

});
```

I do not currently have time to fix this defect, but decided to post an issue in case someone can get to it before I have time.

**Comment by vlucas** [comment]
In your code example, `status != 999`, so the `.expectNot('status', 999)` statement is true, thus the test does not fail. Switch it to `.expectNot('status', 200)` and it will fail.

**Comment by psaxton** [comment]
Sorry, I copy/pasted wrong. Using `.expectNot('status', 200)` still fails for me.

**Comment by vlucas** [comment]
Sadly, you are correct. I am actively looking into this now. :(

**Comment by vlucas** [comment]
This was fixed in 10755b6
